### ISSUE TITLE
demo login

### ISSUE THREAD

**Issue by alirz23** [TryGhost/Ghost#6026]
you guys have a demo up on http://demo.ghost.io but no credentials how does one login to the admin I tried to install it on my local machine took 2 hours to setup the whole thing

**Comment by ErisDS** [comment]
Hi @alirz23, demo.ghost.io is a demo of what a Ghost blog looks like. If you want to try it out you can go to ghost.org, sign up and try our Ghost for 14 days - there are no card details required.

**Comment by pabru** [comment]
This is no longer true, since the site now requires credit card details to proceed with the registration.

**Comment by ErisDS** [comment]
Correct, we require card details but the trial is still free. You can also use our CLI tool to do a fast install - here are the docs on all the ways to get started: https://docs.ghost.org/docs/getting-started-guide

**Comment by pabru** [comment]
I am planning on at least trying out Ghost as my blogging platform, personally speaking, if not actually choosing it out of the other alternatives. I have no trouble with server admin and have a few years of Linux experience.

But, there are non-technical users who will want to try out the full experience, including the admin panel. In their position, I'd be conflicted about giving my credit card number to a website I'm unsure about. Also consider that some people simply do not have any type of payment card (teenagers without a bank account, for one).

I've tried to find a fully-accessible demo install of Ghost, since almost every single PHP-based CMS under the sun has multiple auto-reinstalling demo sites that let you play with them in a sandbox. As far as I can tell, for Ghost it's either the original website (which is non-functional unless you supply CC details), or deploying yourself (which non-technical people won't be able to do).

**Comment by CurtisBelt** [comment]
Just blunt marketing feedback here --- I just installed Ghost locally. Problem is I use node 9.8 which is unsupported. And that's fine, I crossed my fingers and hoped it would work well enough to check out admin page. It didn't work, and I assume it will if I abide by the recommended versions, but let's face it I'm trying to research 5 different products and while I could get into NVM or a different solution, how many minutes have I now spent trying to get a glimpse at your admin panel? Too many.

+1 to this

**Comment by kirrg001** [comment]
@CurtisBelt 

https://docs.ghost.org/docs/install-local 

It describes everything you need. It's a very simple **three** step installation to get a local blog working.

- check supported node versions
- install the CLI
- install a local blog

**If there is something unclear in the docs - please either tell us in slack or suggest a edit.**

**Comment by CurtisBelt** [comment]
@kirrg001 

I appreciate your response however I did not intend to say your docs were unclear. I was just backing up what pabru said https://github.com/TryGhost/Ghost/issues/6026#issuecomment-356239994.

Any bumps in installation (even if it's my own fault) can result in someone skipping Ghost as an option. That's why many other CMS' have the admin dashboard readily available for testing.

**Comment by j4p3** [comment]
(This may not necessarily be a github-relevant issue, as it's more about the platform's marketing materials than about the CMS code. If there is a more relevant location for this post, please feel free to direct me there.)

Chiming in to add that it would be extremely helpful to be able to visit a public URL running the Ghost admin, or at least its text editor. Of course allowing anyone to save posts/content on a public instance would be disastrous, so perhaps persistence would be disabled there.

The ghost local installation process was a significant barrier to me in selecting ghost as a platform. Not necessarily a technical barrier - it's just an npm command and a ghost-cli command - but even after completing the process, there is no indication in the `install local` docs or CLI output of how to log into the newly created local instance. I checked out the source to discover that a new site's admin is open at `/ghost`.

tl;dr Platform looks great! This is a roadblock.

**Comment by ErisDS** [comment]
We do now have a better place for this sort of discussion -> https://forum.ghost.org.

However, the lack of admin URL output is tracked here: https://github.com/TryGhost/Ghost-CLI/issues/665 I will chase getting it sorted out.

**Comment by BubbaJoe** [comment]
Gonna save you guys some time.

Just use Stapi!
